Cargo.toml

```diff
@@ -14,8 +14,9 @@ crate-type = ["cdylib"]
 pyo3-build-config = "0.19.1"
 
 [dependencies]
+base64 = "0.21"
 chrono = "0.4.23"
-tantivy = "0.20.1"
+tantivy = { git = "https://github.com/quickwit-oss/tantivy.git" }
```

Comment:
This is because there hasn't been a release with lenient parser support in `tantivy` yet.